components/create_team/components/display_name.jsx

```diff
@@ -50,10 +61,11 @@ export default class TeamSignupDisplayNamePage extends React.Component {
             return;
         }
 
-        this.props.state.wizard = 'team_url';
-        this.props.state.team.display_name = displayName;
-        this.props.state.team.name = cleanUpUrlable(displayName);
-        this.props.updateParent(this.props.state);
+        const newState = this.props.state;
+        newState.wizard = 'team_url';
+        newState.team.display_name = displayName;
+        newState.team.name = cleanUpUrlable(displayName);
+        this.props.updateParent(newState);
     }
 
     handleFocus(e) {
```

Comment:
I think this should bind to work properly like `handleFocus = (e) => {...}`

Reply:
Ah I can't believe I missed that one, will fix that one right away!